docs/products/kafka/howto/use-kafka-topic-catalog.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Manage Apache Kafka topics with topic catalog
+limited: true
+---
+
+The Aiven for Apache Kafka® topic catalog offers a user-friendly interface to manage your Apache Kafka topics within your Aiven for Apache Kafka services.
+
+## Access topic catalog
+
+1. Log in to the [Aiven console](https://console.aiven.io/).
+1. Click **Tools** in the top menu.
+1. Select **Kafka topic catalog**.
```

Comment:
I've just renamed this in a PR as 'Apache Kafka topic catalog', is 'Kafka topic catalog' correct?